lib/tpm2_attr_util.c

```diff
@@ -495,6 +495,7 @@ static bool common_strtoattr(char *attribute_list, void *attrs, dispatch_table *
 
 bool tpm2_attr_util_nv_strtoattr(char *attribute_list, TPMA_NV *nvattrs) {
 
+    memset(nvattrs, 0, sizeof(*nvattrs));
```

Comment:
@williamcroberts do you think these two first commits should also go to the 3.X stable branch?

Reply:
Probably